ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -407,9 +407,18 @@ StatusDialog {
                         }
                         return qsTr("Swap")
                     }
-                    tooltip.text: root.swapAdaptor.validSwapProposalReceived &&
-                                  root.swapAdaptor.swapOutputData.approvalNeeded ?
-                                      qsTr("Approve %1 spending cap to Swap").arg(fromTokenSymbol) : ""
+                    tooltip.text: {
```

Comment:
Duplicated logic in the `text` binding above. WDYT of using `States`, or extract the logic in a property flag?

Reply:
would prefer doing it in another PR then, this was bug fix for new release and dont want to introduce risk of adding anymore bugs :)